e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/validation/company-details-phone-number-validation.spec.js

```diff
@@ -0,0 +1,127 @@
+      companyDetails.companiesHouseSearch().clear().type(COMPANIES_HOUSE_NUMBER);
+      yesRadioInput().eq(0).click();
+      yesRadioInput().eq(1).click();
+      companyDetails.companyWebsite().clear().type(VALID_WEBSITE);
+      companyDetails.phoneNumber().clear().type(LONG_PHONE_NUMBER);
+      submitButton().click();
+      partials.errorSummaryListItems().should('have.length', 1);
+      partials.errorSummaryListItems().first().invoke('text')
+        .then((text) => {
+          expect(text.trim()).equal(errorMessage);
+        });
+    });
+
+    it('should focus to the phone number section when clicking the error', () => {
+      partials.errorSummaryListItemLinks().first().click();
+      companyDetails.phoneNumber().should('have.focus');
+    });
+
+    it('should display the validation error for phone number', () => {
+      companyDetails.phoneNumberError().invoke('text')
+        .then((text) => {
+          expect(text.trim()).equal(`Error: ${errorMessage}`);
+        });
+    });
+
+    it('should display validation errors if phone number is international number', () => {
+      companyDetails.companiesHouseSearch().clear().type(COMPANIES_HOUSE_NUMBER);
+      yesRadioInput().eq(0).click();
+      yesRadioInput().eq(1).click();
+      companyDetails.companyWebsite().clear().type(VALID_WEBSITE);
+      companyDetails.phoneNumber().clear().type(INTERNATIONAL_PHONE_NUMBER);
+      submitButton().click();
+      partials.errorSummaryListItems().should('have.length', 1);
+      partials.errorSummaryListItems().first().invoke('text')
+        .then((text) => {
+          expect(text.trim()).equal(errorMessage);
+        });
+    });
+
+    it('should display the validation error for phone number', () => {
+      companyDetails.phoneNumberError().invoke('text')
+        .then((text) => {
+          expect(text.trim()).equal(`Error: ${errorMessage}`);
+        });
+    });
+  });
+
+  describe(`when ${PHONE_NUMBER} is left empty`, () => {
+    it('should not display validation errors', () => {
+      companyDetails.companiesHouseSearch().clear().type(COMPANIES_HOUSE_NUMBER);
+      yesRadioInput().eq(0).click();
+      yesRadioInput().eq(1).click();
+      companyDetails.companyWebsite().clear();
+      companyDetails.phoneNumber().clear();
+      submitButton().click();
+      partials.errorSummaryListItems().should('have.length', 0);
+    });
+  });
+
+  describe(`when ${PHONE_NUMBER} is correctly entered`, () => {
+    it('should not display validation errors', () => {
+      companyDetails.companiesHouseSearch().clear().type(COMPANIES_HOUSE_NUMBER);
+      yesRadioInput().eq(0).click();
+      yesRadioInput().eq(1).click();
+      companyDetails.companyWebsite().clear().type(VALID_WEBSITE);
+      companyDetails.phoneNumber().clear().type(VALID_PHONE_NUMBER);
+      submitButton().click();
+      partials.errorSummaryListItems().should('have.length', 0);
+    });
+  });
```

Comment:
i thought there was a lot more possible phone numbers listed in figma, should we be testing the possibilities? Also do we need to test mobile number?